### PR TITLE
feat: introduce observation generation for stunreachability, urlgetter, browser_web

### PR DESCRIPTION
**Pull request by DecFox**
This diff introduces observation generation support for `stun_reachability`, `urlgetter` and `browser_web` nettests.

### CODE REVIEW

File: oonidata/transforms/nettests/browser_web.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+class BrowserWebTransformer(MeasurementTransformer):
+    def make_observations(self, msmt: BrowserWeb) -> Tuple[List[WebObservation]]:
+        bw_obs = WebObservation(
+            **dataclasses.asdict(self.measurement_meta)
+        )
+        
+        bw_obs.http_failure = msmt.test_keys.result
+        bw_obs.http_runtime = msmt.test_keys.load_time_ms
```
Comment (hellais) [inline oonidata/transforms/nettests/browser_web.py:15]: 💯

**Review by hellais — APPROVED** [review]
This looks great, thanks for putting it together!

**Comment by hellais** [comment]
I think I broke the tests as part of the merge commit.

@DecFox can you look into fixing the tests?

**Comment by DecFox** [comment]
Pushed the fixes. Tests are passing now.

**Pull request merged**